diag_manager/diag_util.F90

```diff
@@ -637,6 +637,7 @@ SUBROUTINE log_diag_field_info(module_name, field_name, axes, long_name, units,&
     CHARACTER(len=*), INTENT(in) :: module_name !< Module name
     CHARACTER(len=*), INTENT(in) :: field_name !< Field name
     INTEGER, DIMENSION(:), INTENT(in) :: axes !< Axis IDs
+    CHARACTER(len=*), INTENT(in) :: axes_list !< Comma seperated list of axes names
```

Comment:
did merging not go right or did you add this?

Reply:
Hmm, `git merge mixedmode` didn't catch that for some reason. `mixedmode` version of `diag_util.F90` manually checked out in 26430c02917e2ffb60b66de04f3a5b98d89981bd.